../../pyodbc.pyi,sha256=lVTrjquwnlBRKvppPS-CifIDyFRZDkx03RGjDZo_rXo,13102
pyodbc-4.0.32.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
pyodbc-4.0.32.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
pyodbc-4.0.32.dist-info/METADATA,sha256=UXVCMR-bEEIK2HPIj62wiTsogopWW_yjL_mJzzUXamU,2283
pyodbc-4.0.32.dist-info/RECORD,,
pyodbc-4.0.32.dist-info/REQUESTED,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
pyodbc-4.0.32.dist-info/WHEEL,sha256=fVcVlLzi8CGi_Ul8vjMdn8gER25dn5GBg9E6k9z41-Y,100
pyodbc-4.0.32.dist-info/direct_url.json,sha256=zUW6CaMU68KZUEu6lzXKKIeZYO9qsWN4jwmRwT4gdtI,66
pyodbc-4.0.32.dist-info/top_level.txt,sha256=30jvYcCul6_wA26OT5c068GAHzqQLpd9WfQ0mSSsDTU,7
pyodbc.cp39-win_amd64.pyd,sha256=-K4xtLpV6ZJ1Y2u5CfwYCt4DNO8SOfMoTwvh7B-VuQU,153088
